docs/more-pushmsg.md

```diff
@@ -33,3 +33,53 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
+    "title"=> "Hello World",
+    "message"=> "Test push From PHP cURL.",
+    "priority"=> 5,
+);
+
+$data_string = json_encode($data);
+
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array(
+    'Content-Type: application/json; charset=utf-8'
```

Comment:
Could you use either single or double quotes over the whole php block for consistency?